src/bb-components-build.ts

```diff
@@ -306,7 +344,7 @@ void (async (): Promise<void> => {
 
     type BuildPrefabComponent = BuildPrefab | PrefabPartial;
 
-    const buildPrefabs = prefabs.map((prefab) => {
+    const buildPrefab = (prefab: Prefab): Prefab => {
```

Comment:
We might want to introduce different `Prefab` types for page prefabs, component prefabs and partial prefabs. I've send you a message via slack.